doc/guides/encoding/jpeg_2k.rst

```diff
@@ -32,7 +32,7 @@ with the *Pixel Data* see the :doc:`glossary of Image Pixel elements<../glossary
 | *Samples   | *Photometric          | *Pixel          | *Planar        | *Bits      | *Bits   |
 | per Pixel* | Interpretation*       | Representation* | Configuration* | Allocated* | Stored* |
 +============+=======================+=================+================+============+=========+
-| 1          | MONOCHROME1           | 0 or 1          | (absent)       | 8, 16, 24, | 1 to 38 |
+| 1          | MONOCHROME1           | 0 or 1          | (absent)       | 1, 8, 16,  | 1 to 38 |
```

Comment:
24 has disappeared

Reply:
resolved in [c1e9a0f](https://github.com/pydicom/pydicom/pull/2216/commits/c1e9a0f2c1b8225b608765a11be870fdd5d7fc99)